### PR TITLE
problem: no rfc for channel pattern

### PR DESCRIPTION
**Pull request by somdoron**

### CODE REVIEW

**Comment by somdoron** [comment]
I still need to update ZMTPv3.1 with the new socket type. I will wait after the other PRs are merged and will add the changes.

File: content/docs/rfcs/52/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+The channel pattern is the thread-safe alternative of the exclusive pair pattern.
+CHANNEL is not a general-purpose socket but is intended for specific use cases where the two peers are architecturally stable. This usually limits CHANNEL to use within a single process, for inter-thread communication.
+
+In order for the API to be thread-safe sending and receiving messages MUST be atomic and a single API call to receive or send entire msg. Therefore, the client-server pattern (and the rest of the thread-safe family) MUST NOT allow multipart messages.
+
+## The CHANNEL Socket Type
+
+General behavior:
+
+* MAY be connected to at most one CHANNEL peers, and MAY both send and receive messages.
```
Comment (Wambou) [inline content/docs/rfcs/52/README.md:37]: I would split this line in two.
Each "MAY" should be a separated line to ease checking behaviour

File: content/docs/rfcs/52/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+name: ZeroMQ Channel
+status: Draft
+editor: Doron Somech <somdoron@gmail.com>
+---
+
+This document specifies the semantics of the ZeroMQ channel pattern, which covers the CHANNEL socket type. This specification is intended to guide implementations of this socket type so that users can depend on reliable semantics.
+
+## Preamble
+
+Copyright (c) 2013 iMatix Corporation.
```
Comment (sappo) [inline content/docs/rfcs/52/README.md:13]: Is the copyright correct?

File: content/docs/rfcs/52/README.md (hunk 1 above)
Comment (somdoron) [reply content/docs/rfcs/52/README.md]: fixed

File: content/docs/rfcs/52/README.md (hunk 2 above)
Comment (somdoron) [reply content/docs/rfcs/52/README.md]: fixed

File: content/docs/rfcs/52/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+This specification is intended to formally document the names and expected behaviour of the CHANNEL socket type, which forms the ZeroMQ channel pattern. Conforming implementations of this socket type SHOULD respect this specification, thus ensuring that applications can depend on predictable behavior. This specification is not transport specific, but not all behaviour will be reproducible on all transports.
+
+## Overall Goals of this Pattern
+
+Channel pattern is member of a new family of thread-safe sockets.
+The channel pattern is the thread-safe alternative of the exclusive pair pattern.
+CHANNEL is not a general-purpose socket but is intended for specific use cases where the two peers are architecturally stable. This usually limits CHANNEL to use within a single process, for inter-thread communication.
+
+In order for the API to be thread-safe sending and receiving messages MUST be atomic and a single API call to receive or send entire msg. Therefore, the client-server pattern (and the rest of the thread-safe family) MUST NOT allow multipart messages.
```
Comment (sappo) [inline content/docs/rfcs/52/README.md:31]: message instead of shortcut msg

File: content/docs/rfcs/52/README.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+## The CHANNEL Socket Type
+
+General behavior:
+
+* MAY be connected to at most one CHANNEL peers.
```
Comment (sappo) [inline content/docs/rfcs/52/README.md:37]: peer instead of peers

File: content/docs/rfcs/52/README.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+* MAY both send and receive messages.
```
Comment (sappo) [inline content/docs/rfcs/52/README.md:38]: Suggestion:
 * MAY both send and receive messages in any order.

**Pull request merged**